### DISCUSSION TITLE
Height parameter not working

### DISCUSSION THREAD

**Discussion by zbych232** [andrewsuzuki/react-twitter-widgets#64] [Q&A]
I suppose that the height parameter is not working in Timeline. I can see this behavior in your example [here](https://andrewsuzuki.github.io/react-twitter-widgets/?path=/story/timeline--profile-smaller) and in my app as well. Is it a bug? 
`<Timeline
  dataSource={{
    sourceType: 'profile',
    screenName: 'TwitterDev'
  }}
  options={{
    height: '400'
  }}
/>`

**Comment by andrewsuzuki** [comment]
Hi, this is just a thin wrapper over the official Twitter for Websites. Indeed height isn't working anymore, I'm not sure if it's a bug on Twitter's end or if they've deprecated the parameter. It's still in their documentation [here](https://developer.twitter.com/en/docs/twitter-for-websites/javascript-api/guides/scripting-factory-functions) at the bottom of the page, but I noticed in the developer forum that they may have deprecated some parameters recently:

https://twittercommunity.com/t/new-timeline-embed-is-not-respecting-documented-parameters/175729

So I'd say wait for a bit for a response on the forum
